Utils/src/GenParticlesProducer.cc

```diff
@@ -219,6 +219,7 @@ enum particle_type
     sbottom2=2000005,
     stop2=2000006,
     //HV particles
+    hvscalar=4900001,
```

Comment:
Should we also preemptively add particle_type enums for multiple scalars (4900001-4900006)?

Reply:
Right now, I don't thing we need to. These enums are currently only used for flagging the start and end of a PDGID range to be stored (line 377-378 of the same file), so unless futures users add algorithms that required more sophisticated decay topology crawling, having just 4900001 to mark the staring point should be enough.